.github/workflows/daily.yml

```diff
@@ -28,3 +28,22 @@ jobs:
       - uses: actions-rs/audit-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  notifications:
+    # if: github.ref == 'refs/heads/master'
+    if: always()                              
```

Comment:
what's the reason for that?

Reply:
We want it to run no matter what  - success or failure, and the particular steps will pick the exact conditions.